data/en/products.yml

```diff
@@ -0,0 +1,15 @@
+############################# Products ###################################
+products:
+  enable: true
+  title: Products
+  subtitle: With deep industry and government experience, we can deliver on your most important priorities.
```

Comment:
How about this for the subtitle:

We know it can be tricky to turn research code into a user-facing product, and we're happy share our product expertise, tricks, and lessons learned with our clients.